rosbag2_cpp/include/rosbag2_cpp/cache/message_cache_circular_buffer.hpp

```diff
@@ -45,32 +46,31 @@ namespace cache
 * size.
 */
 class ROSBAG2_CPP_PUBLIC MessageCacheCircularBuffer
+  : public rosbag2_cpp::cache_interfaces::BaseCacheBufferInterface
```

Comment:
Perhaps it could be shortened as:
```suggestion
  : public CacheBufferInterface
```